.github/workflows/release.yml

```
@@ -0,0 +1,148 @@
+          else
+            # For stable releases, explicitly use latest tag
+            NPM_TAG="latest"
+          fi
+          echo "📦 Publishing to NPM with tag: $NPM_TAG"
+          npm publish --access public --tag "$NPM_TAG"
+        env:
+          NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
+
+      - name: Create GitHub Release (production releases only)
+        if: github.event_name == 'push'
+        uses: softprops/action-gh-release@v1
```

actionlint (1.7.7)

the runner of "softprops/action-gh-release@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)